DeploymentExperiences.md

```diff
@@ -0,0 +1,24 @@
+### Deployment Experience
+* Steps 1 - 3: Registering, installing, and logging into Heroku/Heroku CLI were all very straightforward. No issues there.
+* Step 4. Again, navigating to my project/repo directory is muscle memory at this point.
+* Step 5. Installation was, at first, very simple. I forgot to first `pipenv shell` but once I corrected that everything installed without issue.
+* Step 6.
+    > * i. We've worked with `.env` before so initializing was simple.
+    > * ii. Everything looked fine here!
+    > * iii. As suggested, running in a repl worked without issue.
+    > * iv. This was where I got hung up the longest. For whatever reason, despite confirmation of successful installation, `python-decouple` and `dj-database-url` simply would not be recognized within my project. By updating `npm` and VSCode as well as installing Windows Update, I finally got it working properly although I'm still not sure what the correct fix was.
```

Comment:
This can sometimes be a pretty time consuming process, since you don't always think to try updating global packages, etc., and instead look for nonexistent issues in your code.